test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3674,9 +3674,12 @@ def forward(self, x):
         self.run_test(Model(), x)
 
     def test_layer_norm(self):
-        model = torch.nn.LayerNorm([10, 10])
-        x = torch.randn(20, 5, 10, 10)
-        self.run_test(model, x)
+        # As layer_norm works on the last D dimension, please keep
+        # this test case at least three dimension to prevent the
+        # situation of axis=2 mapping to the same axis as axis=-2
+        model2 = torch.nn.LayerNorm([10, 10, 10])
```

Comment:
Any reason this is named model2? For clarity we can just do `layer_norm_model = ...` or just `model`

Reply:
Ahh, I was going to put two tests, and then only left one. I will clean it, and figure out the CI failure with beartype.